CHANGES.rst

```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+2.0.9 (2016-05-1)
+------------------
```

Comment:
Please fix the changelog header, you need to add your fix description under the ``2.0.9 (unreleased)`` header